app/src/main/java/com/example/stretchy/features/executetraining/sound/data/NotifyEvent.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.stretchy.features.executetraining.sound.data
+
+import com.example.stretchy.repository.Activity
+
+
+sealed class NotifyEvent {
```

Comment:
**NotifyEvents**  does not say anything about subject, i mean events of what. Notify brings no information, notify about what event. It has to be more precise, maybe trainingEvents or something other. 
Other than that, other names are good.